lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -61,12 +61,12 @@
                 <table role="presentation" width="100%" align="left" border="0" cellpadding="0" cellspacing="0">
                     <tr>
                         <td width="70">
-                            <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://courses.xpro.mit.edu/static/mitxpro-theme/images/logo.png" width="130"
+                            <a href="{% with_link_tracking site_configuration_values.HOMEPAGE_URL %}"><img
+                                    src="https://mitxonline.mit.edu/static/images/mitx-online-logo.png" width="135"
```

Comment:
I see you followed point#1 from my comment https://github.com/mitodl/mitxonline-theme/pull/25#discussion_r718274241.

But, I still see a contracted icon in the email. Probably, you want to set width to `auto` I'll add the screenshot with both 135px and auto so that you can take a better look at what I'm saying.

**When `Width="auto"`**
<img width="319" alt="Screenshot 2021-09-30 at 4 12 42 PM" src="https://user-images.githubusercontent.com/34372316/135447474-dc66f309-4b6f-4ef5-946d-c3bf64486506.png">

**When `Width="135"`**
<img width="222" alt="Screenshot 2021-09-30 at 4 12 56 PM" src="https://user-images.githubusercontent.com/34372316/135447575-0fd27870-3d57-4d47-834f-5c24eea7ebb5.png">